src/components/CreateCourseRunPage/index.jsx

```diff
@@ -29,37 +27,32 @@ class CreateCourseRunPage extends React.Component {
   componentDidMount() {
     const {
       courseInfo,
+      courseOptions,
+      courseRunOptions,
+      fetchCourseInfo,
+      fetchCourseOptions,
+      fetchCourseRunOptions,
       id,
     } = this.props;
-    if (!courseInfo ||
+    if (!!courseInfo ||
```

Comment:
You're inverting this check? I dunno if that makes sense, since `{}` is always truthy, so this bit isn't doing anything useful anymore. You can probably just drop it altogether and just have the `entries.length` check. Same below.

Reply:
Right. It used to default to `null` so I think the `!courseInfo` handled it. I've updated the logic to check for `courseInfo.data` since leaving just the `entries.length` check fails if the object does not have `courseInfo.data` defined (and our default value is `{}`). Alternatively, might be able to default to `courseInfo = { data: {} }`, but I'm not sure what is better